server/concurrent/semaphore.go

```diff
@@ -0,0 +1,42 @@
+package concurrent
```

Comment:
http://www.golangpatterns.info/concurrency/parallel-for-loop

Pas encore utilisé à l'heure actuelle. Ce sera pour paralleliser les delete de tag dans toute les collections. Mais on peut pas encore le faire parce qu'il faut copier la session Mongo. Et le module n'est pas fait pour, il y a un peu de refacto à faire d'abord.

Mais j'ai vu qu'en fait on pourrait utiliser `sync.WaitGroup` à la place.